### ISSUE TITLE
Restore emblems and engraving bench

### ISSUE THREAD

**Issue by liach** [Railcraft/Railcraft#890]
We need to restore the emblems in future versions. So need we add back engraving bench.
Ref #1373, the item of engraving bench will be handled by this issue instead.
Note: We can utilize TileEntityItemStackRenderer to render emblems.

Old topic:
The current emblems are hosted on some website that cannot be accessed from countries like China. When I try to unlock emblems on servers in China, I cannot unlock those emblems. I hope you could get another better alternative server for emblem hosting.

**Comment by CovertJaguar** [comment]
You can't access dropbox? What about railcraft.info?

**Comment by liach** [comment]
https://www.quora.com/Is-Dropbox-blocked-in-China-Why

**Comment by liach** [comment]
Especially on servers hosted in China it is pretty much unrealistic to use a proxy for dropbox only traffics. railcraft.info is okay though, and it's actually pretty fast to connect as I tried. Other sites, like curse, enjin would be slow. Twitter and google are also blocked.

**Comment by liach** [comment]
Which server do you play Railcraft actually? It'd be cool if players like me could see the author of Railcraft, yet people like to use Railcraft to build subway systems.
Some screenshots:
![qq 20160911143331](https://cloud.githubusercontent.com/assets/7806504/18420678/17708768-782d-11e6-94ff-7f8d9dc38bd2.jpg)
![qq 20160911143754](https://cloud.githubusercontent.com/assets/7806504/18420686/5ddc89d6-782d-11e6-889e-d21d3b0a9719.png)

**Comment by liach** [comment]
![image](https://cloud.githubusercontent.com/assets/7806504/18420756/e7691cc2-782e-11e6-99a5-68fffb5de0dc.png)
By the way @CovertJaguar this is what I found after applying your ide settings in the repository. I actually cannot see the password though. I hope you can fix this...

**Comment by CovertJaguar** [comment]
Hmm..

**Comment by CovertJaguar** [comment]
My net is so bad is not really possible for me to play on servers for long periods.

**Comment by CovertJaguar** [comment]
I don't actually see any passwords in the files.

**Comment by liach** [comment]
I believe that we might need to have a chat on irc

**Comment by liach** [comment]
May it be possible to host emblems on railcraft.info?

**Comment by CovertJaguar** [comment]
Should be, I can look into mirroring them there for 1.10.

**Comment by liach** [comment]
Probably also for versions for 1.7 after the official releases of 1.10

**Comment by 3TUSK** [comment]
Question: last time when I searched for available public emblem code(s), I only found about 3~4 codes. Are those all?

**Comment by liach** [comment]
youtube dark tower

**Comment by 3TUSK** [comment]
No time to go through Dark Tower seires...

**Comment by liach** [comment]
@3TUSK Just see screenshots of different parts of the video, typically in the end or in middle.

**Comment by 3TUSK** [comment]
gotacha

**Comment by CovertJaguar** [comment]
Watch the episode 100(ish) world tour.

**Comment by liach** [comment]
@CovertJaguar I think you should try to add this back since it is a point of interest of users, at least more attractive than decorative blocks.

**Comment by CovertJaguar** [comment]
Rendering is completely broken atm. I've barely even looked at what needs to be fixed.

**Comment by liach** [comment]
What's worse, dropbox broke all the links. We need to find a way to fix that

**Comment by liach** [comment]
Followup: @forecaster would be glad to host the emblems.

**Comment by CovertJaguar** [comment]
Emblems are uploaded to the new location. We just need to update the link.

Is rendering still broken?

**Comment by liach** [comment]
Yes. The model should be simple; online grabbing textures should be easy, too. In the worst case, we can still tile entity item stack renderers.
